osivalidator/osi_trace.py

```diff
@@ -42,9 +41,11 @@ def get_size_from_file_stream(file_object):
 class OSITrace:
     """This class wrap OSI data. It can import and decode OSI traces."""
 
-    def __init__(self, show_progress=True, path=None, type_name="SensorView"):
+    def __init__(self, show_progress=True, path=None, type_name="SensorView", buffer_size=1000000):
```

Comment:
It is not a very good style to have magic numbers in the code, please move stuff like this into a config or global structure/ config class

Reply:
Good point!

Reply:
Just make a Singleton Dataholder class for the configuration parameter

Reply:
I added those numbers into the cli arguments see the latest commit